qcoptions.py

```diff
@@ -21,4 +21,9 @@
                   help="[OPTIONAL] Run a shortened test after covers have been applied")
 parser.add_option("--ztrim", type="float", dest="ztrim", default=0.0,
                   help="Specify the p value of the trim", metavar="ztrim")
-
+parser.add_option("--scanmin", type="int", dest="scanmin",
```

Comment:
Depending on the scan type, you don't necessarily want to scan the same range, and having these parameters removes the worse practice of hard-coding values